pkg/rollout/rollout.go

```diff
@@ -0,0 +1,191 @@
+package rollout
+
+import (
+	"fmt"
+
+	runapi "github.com/GoogleCloudPlatform/cloud-run-release-operator/internal/run"
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+	format "github.com/logrusorgru/aurora"
+	"github.com/pkg/errors"
+	"google.golang.org/api/run/v1"
+)
+
+// Logger describes a logger printing information to the user.
+type Logger interface {
+	Println(args ...interface{})
+	Printf(format string, args ...interface{})
+}
+
+// Rollout is the rollout manager.
+type Rollout struct {
+	RunClient        runapi.Client
+	Config           *config.Config
+	Log              Logger
+	promoteCandidate bool
+}
+
+// Automatic tags.
+const (
+	StableTag    = "stable"
```

Comment:
Do you prefer `StableTag`, `CandidateTag` or `TagStable`, `TagCandidate`?

Reply:
this is good